packages/contracts/src/NounsToken.sol

```diff
@@ -161,17 +161,20 @@ contract NounsToken is INounsToken, Ownable, ERC721Checkpointable {
      * starting at 0, until 17520 lil nounder Nouns have been minted (5 years w/ 24 hour auctions).
      * starting at 0, until 8760 lil nounder Nouns have been minted (5 years w/ 24 hour auctions).
      * @dev Call _mintTo with the to address(es).
+     *
+     * @dev Adds blockNumber parameter to minting function.
+     * Necessary to ensure that the seed for Nouns minted in past blocks from the VRGDA pool are correct.
      */
-    function mint() public override onlyMinter returns (uint256) {
+    function mint(uint256 blockNumber) public override onlyMinter returns (uint256) {
         if (_currentNounId <= 175300 && _currentNounId % 10 == 0) {
-            _mintTo(lilnoundersDAO, _currentNounId++);
+            _mintTo(lilnoundersDAO, _currentNounId++, blockNumber);
```

Comment:
@IAm0x52 is it OK to use the blockNumber here for the nounder mints as well?

Reply:
Yeah this is actually the best solution. Since block can only be used once this is more correct than using block.number - 1.